dbms/src/Interpreters/WindowDescription.h

```diff
@@ -61,11 +61,11 @@ struct WindowFrame
     FrameType type = FrameType::Ranges;
 
     BoundaryType begin_type = BoundaryType::Unbounded;
-    Field begin_offset = Field(UInt64(0));
+    UInt64 begin_offset = 0;
```

Comment:
where is `begin_offset` initialized?

Reply:
> begin_offset

`WindowDescription::setWindowFrame` is responsible for initializing it and it is called in `DAGExpressionAnalyzer::buildWindowDescription`.